src/compile.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import fs from 'node:fs';
-import path from 'node:path';
+import fs from 'fs';
+import path from 'path';
```

Comment:
This change was made to avert this error: `Cannot find module 'node:fs'` which happens when `yarn build` is run